File: pertemuan 6/Tugassementara.py
print("........Sistem Siakad.......")
print("1 . Tambah data")
print("2 . Edit data")
print("3 . Hapus data")
print("4 . Lihat data")

Menu = input("Menu yang ada pilih :")
Menu = int(Menu)

if ( Menu == 1):
    print()
    print("1. Tambah data Mahasiswa")
    print("2. Tambah data mata kuliah")
    print("3. Kembali")
    
    Menu1 = input("Menu yang anda pilih")
    Menu1 = int(Menu1)
    if (Menu1 == 1):
        print()
        nama = input("Masukan nama Anda")
        nama = str(nama)
        print("Nama Anda Adalah" + nama)
    elif (Menu1 == 2):
        print()
        mataKuliah = input("Masukkan mata Kuliah yang ingin ditambahkan")
        mataKuliah = str(mataKuliah)
        print("Mata kuliah yang ingin ditambahkan adalah" + mataKuliah)
    elif (Menu1 == 3):
        print()
        print("......Terima kasih......")
    else:
       print("Menu tidak ada")
elif(Menu == 2 or 3 or 4):
    print("Fitur belum ada")

else:
    print("Menu tidak ada")


    
    

File: pertemuan 6/Tugassementara.java
import java.util.Scanner;

public class Tugassementara {
    public static void main(String[] args) {
        System.out.println(".....Sistem siakad.....");
        System.out.println("1 . Tambah data");
        System.out.println("2 . Edit data");
        System.out.println("3 . Hapus data");
        System.out.println("4 . Lihat data");
        
        Scanner console = new Scanner(System.in);
        System.out.print("Menu yang ada pilih :");
        int Menu = console.nextInt();

        if (Menu == 1) {
            System.out.println("1. Tambah data Mahasiswa");
            System.out.println("2. Tambah data mata kuliah");
            System.out.println("3. Kembali");

            System.out.print("Menu yang anda pilih");
            int Menu1 = console.nextInt();
            console.nextLine();
             if (Menu1 == 1) {
                 System.out.println();
                 System.out.print("Masukan nama Anda :");
                 String nama = console.nextLine();
                 System.out.println("Nama anda adalah :" + nama);

             } else if (Menu1 == 2) {
                 System.out.println();
                 System.out.print("Masukkan mata Kuliah yang ingin ditambahkan : ");
                 String mataKuliah = console.nextLine();
                 System.out.println("Mata kuliah yang ingin ditambahkan adalah" + mataKuliah);

             } else if (Menu1 == 3) {
                 System.out.println();
                 System.out.println("......Terima kasih......");

             } else {
                 System.out.println("Menu tidak ada");
             }
        }else if (Menu >= 2 && Menu < 5) {
            System.out.println("Fitur belum ada");

        }else {
            System.out.println("Menu tidak ada");
        }
    }
}